frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -90,6 +91,7 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
   );
   const agency = getIn(formikProps.values, withNameSpace('agencyId'));
   const { lookupCodes } = useCodeLookups();
+  const agencies = (props.agencies ?? []).map(c => mapSelectOptionWithParent(c, props.agencies));
```

Comment:
need to preset the agency list so that it the data shows up properly in the parentselect.